sonar-project.properties

```diff
@@ -1,8 +1,8 @@
 sonar.projectKey=edge:devicedb:master
 sonar.projectName=devicedb:master
 sonar.projectVersion=1.0.0
-sonar.host.url=https://pe-jm.usa.arm.com:9000
-sonar.login=80cdc15bd34e45ccb396fc79f23ab454c94b8c1e
+sonar.host.url=http://127.0.0.1:9000
+sonar.login=9d6ebe8dd8b28dcee8195e204fd93d2e2480da8a
```

Comment:
Is this some sort of secret like a password? Is it safe to have it in version control?

Reply:
@jrife Yes this is an account token for SonarQube admin account but sonarqube is not being used as of now and the server is down that is why didn't bother on this. When we bring up the server and integrate it with the CI, we will encrypt this.

Reply:
And this token doesn't work also now as it has been revoked.

Reply:
@sameer2209-arm OK. If it doesn't work, can it be removed?

Reply:
@jrife removed the host url and token in the new commit